### ISSUE TITLE
Accessibility - Missing form label at Attach supporting evidence page

### ISSUE THREAD

**Issue by lmcbhvu** [cds-snc/report-a-cybercrime#2389]
## Summary

Got an error (missing form label) when run WAVE on Attach supporting evidence page

![image](https://user-images.githubusercontent.com/60269104/96627608-1ee0fb80-12df-11eb-9cc4-51becd8da3ae.png)

**Comment by DianeLiu2019** [comment]
error picture
![image](https://user-images.githubusercontent.com/51336707/97193432-60f6bb00-177f-11eb-89b4-87cde222ae6f.png)
